Makefile

```diff
@@ -852,27 +873,27 @@ clean-stress-test-tools:
 clean: clean-images clean-binaries clean-site clean-release clean-build clean-protos clean-swagger-docs clean-nodejs clean-install-yaml clean-stress-test-tools clean-secrets
```

Comment:
Also need to modify clean-binaries and clean-images

Reply:
This was there.